addons/web/static/src/webclient/settings_form_view/fields/upgrade_boolean_field.js

```diff
@@ -34,4 +34,9 @@ export class UpgradeBooleanField extends BooleanField {
     }
 }
 UpgradeBooleanField.isUpgradeField = true;
+UpgradeBooleanField.additionalClasses = [
+    ...UpgradeBooleanField.additionalClasses || [],
```

Comment:
Pretty sure this was meant to be `...BooleanField.additionalClasses` and not `...UpgradeBooleanField.additionalClasses`

Reply:
This was on purpose, as I am adding `o_field_boolean` to the maybe-already defined `additionalClasses` in `BooleanField`.
And in this case, since `UpgradeBooleanField` directly extends `BooleanField`, does it matter much ?

Reply:
Conveying intent is important, while it's semantically equivalent, I think

```js
...(BooleanField.additionalClasses || []),
```

Clearly conveys that you're trying to keep the classes from the original field.

All in all though, it's really not that important, let's merge it like this.